TestApps/TestApp/AppDelegate.swift

```diff
@@ -33,11 +30,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
 
         MobileCore.setLogLevel(.trace)
-        var extensions: [NSObject.Type] = [Media.self]
+        var extensions: [NSObject.Type] = [Media.self, Edge.self, AEPEdgeIdentity.Identity.self]
```

Comment:
we can use just `Identity.self` here since this is the only module dependency installed